src/tsx/liveScreen.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+
+interface IState {
+  times: number;
+}
+
+class LiveScreen extends React.PureComponent<{}, IState> {
+  constructor(props: {}) {
+    super(props);
+    this.state = { times: 0 };
+    this.handleKey = this.handleKey.bind(this);
```

Comment:
Overwriting this.handleKey looks ugly to me, but it seems to be how people are doing it. Am I missing something or is it fine?

Reply:
Looks like the way to do it if you need to use the state from the model.